site/utils/index.ts

```diff
@@ -49,9 +49,56 @@ export const setZodDefaultError: (errorMessage: string) => { errorMap: ZodErrorM
 
 const dateRegex = /^(0?[1-9]|[12][0-9]|3[01])[\/](0?[1-9]|1[012])[\/]\d{4}$/;
 const timeRegex = /^([0-1][0-9]|2[0-3])[0-5][0-9]$/;
+const minutesRegex = /^[0-9]{0,3}$/;
+
+/**
+ * Verify if the input value is of date format DD/MM/YYYY.
+ * @param {defaultError} defaultError Error message when the validation fails.
```

Comment:
Do we need 'defaultError' before the word 'Error' ? Same on the 2 lines below also.

Reply:
If the string is removed then the below data is displayed

![image](https://user-images.githubusercontent.com/95031756/226330308-705c5b1a-7d8c-4255-8b9b-13b573f46b5e.png)

Reply:
I'm not sure what it looks like with 'defaultError' in there too but it doesnt really matter.